### ISSUE TITLE
Patch for /Satelite/main.c

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#506]
```
use config file for recovery module path
```

Original issue reported on code.google.com by `crah...@hotmail.com` on 11 Sep 2012 at 11:23

Attachments:
- [main.c.patch](https://storage.googleapis.com/google-code-attachments/procfw/issue-506/comment-0/main.c.patch)

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Indeed, the code is inconsistent because it is defined in the config file but 
not used in the module.

Thank you for the patch
```

Original comment by `devnonam...@gmail.com` on 28 Aug 2013 at 10:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `devnonam...@gmail.com` on 28 Aug 2013 at 10:07
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 41421afce881
```

Original comment by `devnonam...@gmail.com` on 25 Sep 2013 at 1:50
- Changed state: **Fixed**
